cf-console-header-menu-button-sample/src/aem-cf-console-admin-1/web-src/src/components/TipsModal.js

```diff
@@ -0,0 +1,84 @@
+/*
+Copyright 2024 Adobe. All rights reserved.
+This file is licensed to you under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License. You may obtain a copy
+of the License at http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software distributed under
+the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+OF ANY KIND, either express or implied. See the License for the specific language
+governing permissions and limitations under the License.
+*/
+
+import React, { useState, useEffect } from "react";
+import { attach } from "@adobe/uix-guest";
+import {
+  Provider,
+  Content,
+  defaultTheme,
+  Button,
+  TooltipTrigger,
+  Tooltip,
+  View
+} from "@adobe/react-spectrum";
+import "./TipsModal.css";
+
+import { extensionId } from "./Constants";
+
+export default TipsModal = () => {
+  const [guestConnection, setGuestConnection] = useState();
+
+  const [tipIndex, setTipIndex] = useState(0);
+
+  useEffect(() => {
+    const init = async () => {
+      const connection = await attach({ id: extensionId });
+      setGuestConnection(connection);
+    };
+
+    init().catch(console.error);
+  }, []);
+
+  const onPressUIExtensibilityTipBtnHandler = (e) => {
+    if (tipIndex === 3) {
+      setTipIndex(1);
+    } else {
+      setTipIndex(++tipIndex);
+    }
+  };
+
+  const onCloseHandler = () => {
+    guestConnection.host.modal.close();
+  };
+
+  return (
+    <Provider theme={defaultTheme} colorScheme="light">
```

Comment:
Let's showcase some of the React Spectrum components, like Text, Divider, Well, Button, Toast, etc.

Reply:
Per our discussion I'll show UI Extensibility features.